### PR TITLE
Documentation for dotnet-trace report topN

### PR DESCRIPTION
**Pull request by mikelle-rogers**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by noahfalk — APPROVED** [review]
LGTM

**Review by davmason — APPROVED** [review]
Thanks!

File: docs/core/diagnostics/dotnet-trace.md [hunk 1]
```diff
@@ -253,6 +254,44 @@ Lists pre-built tracing profiles with a description of what providers and filter
 dotnet-trace list-profiles [-h|--help]
 ```
 
+## dotnet-trace report
+
+Generates a report into stdout from a previously generated trace.
```
Comment (gewarren) [inline docs/core/diagnostics/dotnet-trace.md:259]:
```suggestion
Creates a report into stdout from a previously generated trace.
```

Just so generate isn't used twice in the sentence.

File: docs/core/diagnostics/dotnet-trace.md [hunk 2]
```diff
@@ -253,6 +254,44 @@ Lists pre-built tracing profiles with a description of what providers and filter
...
+
+- **`<tracefile>`**
+
+  The file path for the trace being analyzed.
+
+### Commands
+
+#### dotnet-trace report topN
+
+Finds the top N methods on the callstack the longest.
```
Comment (gewarren) [inline docs/core/diagnostics/dotnet-trace.md:277]:
```suggestion
Finds the top N methods that have been on the callstack the longest.
```

File: docs/core/diagnostics/dotnet-trace.md [hunk 3]
```diff
@@ -253,6 +254,44 @@ Lists pre-built tracing profiles with a description of what providers and filter
...
+
+##### Options
+
+- **`-n|--number <n>`**
+
+Gives the top N methods in the callstack.
+
+- **`--inclusive`**
+
+Output the topN methods based on inclusive time. If not specified, exclusive time is used by default.
```
Comment (gewarren) [inline docs/core/diagnostics/dotnet-trace.md:293]:
```suggestion
Output the top N methods based on inclusive time. If not specified, exclusive time is used by default.
```

Might be worth explaining what inclusive means?

**Review by tommcdon — APPROVED** [review]

File: docs/core/diagnostics/dotnet-trace.md [hunk 4]
```diff
@@ -253,6 +254,48 @@ Lists pre-built tracing profiles with a description of what providers and filter
...
+
+##### Options
+
+- **`-n|--number <n>`**
+
+Gives the top N methods on the callstack.
+
+- **`--inclusive`**
+
+Output the top N methods based on [inclusive](https://docs.microsoft.com/visualstudio/profiling/understanding-sampling-data-values?view=vs-2017) time. If not specified, exclusive time is used by default.
```
Comment (gewarren) [inline docs/core/diagnostics/dotnet-trace.md:293]:
```suggestion
Output the top N methods based on [inclusive](/visualstudio/profiling/understanding-sampling-data-values) time. If not specified, exclusive time is used by default.
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**